src/db/tools/services/delete_service.py

```
@@ -0,0 +1,169 @@
+    "delete_pages_users_to_main",
+    "delete_project",
+    "delete_qid",
+    "delete_qid_other",
+    "delete_record_by_pk",
+    "delete_refs_count",
+    "delete_translate_type",
+    "delete_user_page",
+    "delete_user_page_to_main",
+    "delete_views_new",
+    "delete_word",
+]
```

Ruff (0.15.20)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)